lib/scraper/scraper.js

```diff
@@ -4,26 +4,25 @@ Original author: Daniel Biwer (danbiwer@gmail.com)
 */
 
 'use strict';
-var cheerio = require('cheerio');
-var request = require('request');
+const cheerio = require('cheerio');
+const request = require('request');
 
-var util = module.exports = {};
+const util = module.exports = {};
 
 util.processBody = function(body, params){
-  var result = new Object();
-  var pagination = new Object();
-  var listing = new Array();
-  var $ = cheerio.load(body);
-  var paginationBody = $('.pagination.bottom').get();
-  pagination = util.processPaginationResponse(paginationBody, params);
-  if(!$('shortcut_navigable')) listing.push(
-    { title: $('title').text().slice(0, -19), 
+  const result = {};
+  const listing = [];
+  const $ = cheerio.load(body);
+  const paginationBody = $('.pagination.bottom').get();
+  const pagination = util.processPaginationResponse(paginationBody, params);
+  if(!($('.shortcut_navigable').length))
+    listing.push({
+      title: $('title').text().slice(0, -19), 
       thumbnail: $('.thumbnail_center img').eq(0).attr('src')
     });
   else {
     $('.shortcut_navigable').each(function(i, obj){
-      var item = util.processItem(obj);
-      item.thumbnail = $('.thumbnail_center img').eq(0).attr('src');
```

Comment:
Hey Casey - you need line 26 to get the image in object

      item.thumbnail = $('.thumbnail_center img').eq(0).attr('src');